tslint.json

```diff
@@ -15,7 +15,8 @@
         ],
         "max-classes-per-file": false,
         "object-literal-sort-keys": false,
-        "no-console": false
+        "no-console": false,
+        "max-line-length": [true, 140]
```

Comment:
Monitors are wide these days. I think it's defensible, and a better use of real estate. However, we should probably get together to discuss our overall policies on this. There are other changes I'd like to make to our `tslint` files.